pylint/checkers/classes.py

```diff
@@ -765,6 +788,34 @@ def _check_useless_super_delegation(self, function):
             self.add_message('useless-super-delegation', node=function,
                              args=(function.name, ))
 
+    @staticmethod
+    def _check_different_default_values_in_overridden_method(function):
+        """
+        Check if the parameters default values in overridden methods are
+        different from those in the base class
+        """
+        klass = function.parent.frame()
+        for overridden in klass.local_attr_ancestors(function.name):
+            # get astroid for the searched method
+            try:
+                meth_node = overridden[function.name]
+            except KeyError:
+                # we have found the method but it's not in the local
+                # dictionary.
+                # This may happen with astroid build from living objects
+                continue
```

Comment:
this branch is not taken in tests - it would be cool to have this covered by some automated test, but I guess we do not have to be that restrictive.